orix/plot/unit_cell_plot.py

```diff
@@ -0,0 +1,194 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018-2021 the orix developers
+#
+# This file is part of orix.
+#
+# orix is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# orix is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with orix.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Private tools for plotting unit cells given a crystal symmetry and
+orientation.
+"""
+
+from itertools import combinations, product
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from orix.vector import Vector3d
+
+from matplotlib.patches import FancyArrowPatch
+from mpl_toolkits.mplot3d import proj3d
+import numpy as np
+from diffpy.structure import Structure, Lattice
+
+from orix.vector import Vector3d
+
+# taken from SO post https://stackoverflow.com/a/22867877/12063126
+class Arrow3D(FancyArrowPatch):
+    "Matplotlib arrows in 3D."
```

Comment:
These arrows can be useful in the fundamental zone plots as well (see for example the [orientation clustering user guide](https://orix.readthedocs.io/en/stable/clustering_orientations.html#Visualisation)), I suggest moving the class to a private `orix/plot/_util.py` file for now.

```suggestion
class Arrow3D(FancyArrowPatch):
    """Matplotlib arrows in 3D with arrowheads.
    
    Taken from this Stackoverflow answer by user CT Zhu
    https://stackoverflow.com/a/22867877/12063126.
    """
```